server/app/views/applicant/ProgramCardViewRenderer.java

```diff
@@ -56,24 +58,108 @@
 import views.style.ApplicantStyles;
 import views.style.BaseStyles;
 import views.style.ReferenceClasses;
+import views.style.StyleUtils;
 
-/** A renderer to create an individual program information card. */
+/** A renderer to display program information cards individually and in groups. */
 public final class ProgramCardViewRenderer {
   private final ApplicantRoutes applicantRoutes;
   private final ProfileUtils profileUtils;
   private final PublicStorageClient publicStorageClient;
   private final SettingsManifest settingsManifest;
+  private final ZoneId zoneId;
 
   @Inject
   public ProgramCardViewRenderer(
       ApplicantRoutes applicantRoutes,
       ProfileUtils profileUtils,
       SettingsManifest settingsManifest,
-      PublicStorageClient publicStorageClient) {
+      PublicStorageClient publicStorageClient,
```

Comment:
FYI there may be merge conflicts with https://github.com/civiform/civiform/pull/6279 depending on which gets in first.